config/swagger.yaml

```diff
@@ -966,7 +966,8 @@ paths:
         - contract
         - debug
       operationId: PostContractCreateCompute
-      description: 'Compute the call_data for SOPHIA and get a contract_create transaction object'
+      description: '(DEPRECATED to-be-removed in 3.0) - Compute the call_data for SOPHIA and get a contract_create transaction object'
+      deprecated: true
```

Comment:
I see [nicely represented in Swagger UI](http://aeternity.com/api-docs/?config=https://raw.githubusercontent.com/aeternity/aeternity/PT-163858683-deprecate_compiler_api/config/swagger.yaml#/internal/PostContractCreateCompute). 👍